tsconfig.json

```diff
@@ -22,23 +22,24 @@
     "jsx": "react",
     "baseUrl": ".",
     "paths": {
-        "@mattermost/client": ["packages/client/src"],
+        "@mattermost/client/*": ["packages/client/src/*"],
```

Comment:
Changed so that `@mattermost/client/helpers` can be imported - https://github.com/mattermost/mattermost-webapp/pull/11468/files#diff-80257de180e31f91f71bf217a70338b1d792796be24462de1fcb060c16d6d797R20

Reply:
Reverted this change and worked around for not using `buildQueryString`. See https://github.com/mattermost/mattermost-webapp/pull/11468/commits/883848ec32a8e17b5a37f4c6c3a721efd49630d7 for not introducing `createUserX`.